prep.ps1

```diff
@@ -18,6 +18,7 @@ New-Item -force -ItemType directory -Path $mydocuments\Hybrasyl\world\xml\creatu
 New-Item -force -ItemType directory -Path $mydocuments\Hybrasyl\world\xml\items
 New-Item -force -ItemType directory -Path $mydocuments\Hybrasyl\world\xml\itemvariants
 New-Item -force -ItemType directory -Path $mydocuments\Hybrasyl\world\xml\localization
+New-Item -Force -ItemType directory -Path $mydocuments\Hybrasyl\world\xml\lootsets
```

Comment:
Nice job remembering to update this. Can you update the `example.zip` file too?

Reply:
I thought this was to be done via the initial configuration when launching the server? I advise against pushing powershell scripts expecting the users to know how to use them.

Reply:
Example:

```if(!Directory.Exists(MapFileDirectory))
{
  Directory.Creat(MapFileDirectory);
}```

Reply:
Currently it isn't; we say to use the PS1 or unzip the zipfile in the readme. This is to make it easier to start the server for new users.

Reply:
That's my point.. this should be fixed on server initialization rather than using the powershell script.

Reply:
OK - out of scope here as the PS1 went in via one of my PRs. So we should open a github issue to fix.

Reply:
ok that's fine, then that ps file should be removed from the scope of loot fixes then and added as a separate PR.

..or it can just be brought into scope here.

Reply:
Github Ticket Created: #139